### PR TITLE
move plugin under vector menu

### LINKED ISSUE #43: Duplicate menu entries
screen shot 2016-12-23 at 10 07 19 am

### PR DESCRIPTION
**Pull request by meomancer**
this fix #43 
this partial fix for issue #25 
screenshot from 2016-12-27 12 53 50
screenshot from 2016-12-27 13 23 23

### CODE REVIEW

File: cadasta/plugin.py [hunk 1]
```diff
@@ -21,15 +21,17 @@
  ***************************************************************************/
 """
 import logging
-from qgis.PyQt.QtCore import QSettings, QTranslator, qVersion, QCoreApplication
-from qgis.PyQt.QtGui import QAction, QIcon, QMenu, QWidget
+import os.path
+from qgis.PyQt.QtCore import QSettings, QTranslator, qVersion, \
```
Comment (ismailsunni) [inline cadasta/plugin.py]: Avoid the back slash and use bracket '('

**Comment by Gustry** [comment]
* Let's move `icon.png` from the root of the plugin to the resource folder. Just think to change the path in the metadata.txt file.
* And I would be interested to have the cadasta icon in the vector menu ;-)
( I wanted to do the same for InaSAFE)
* Rename the plugin to `Cadasta` only (in the plugin manager). This is from the metadata.

**Comment by Gustry** [comment]
And I would be interested to have the cadasta icon in the vector menu ;-)
( I wanted to do the same for InaSAFE)

**Comment by meomancer** [comment]
updated:
screenshot from 2016-12-28 09 10 10

@Gustry do you know how to give icon in vector menu?
i already searched it and couldn't find it (although now i just can add icon on action) :(

**Comment by Gustry** [comment]
Ok, it shouldn't be a blocker. Let's move on if you want.
i think you add the icon to the action, it will work. I did it in another plugin, I will have a look.
Looks good to me, thanks !

**Pull request merged**